rebac-admin-backend/script/pull-spec.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env sh
```

Comment:
why not using a git submodule?

Reply:
+1

Reply:
That was my first approach, but then I noticed I just need the `openapi.yaml` file and not the rest of the repository.

But, now as I see, we can just drop this copy of the spec and embed it in the auto-generated code (See the `scope` option [here](https://github.com/deepmap/oapi-codegen#using-oapi-codegen)). This way we can even serve that spec, which could be useful for debugging/inspection to see what's is the exact spec that a backend implementers.

@alesstimec @shipperizer Sounds good?

Reply:
ok, this is probably shooting for the starts but i ll drop it anyway

ideally this gets updated automatically at every given update of the underlying source (so the openapi-repo), updated as a GH action runs and creates a PR

codebase is supposed ot be only stable when it s relaeased, so we wouldn't incur in any issue by having this kind of machinery (which is not needed as part of this PR, not asking for that dont worry)

i wouldnt be that bothered about having  a full repo as a submodule even if we used only one file, reason is we might have to offload the makefile to that repo  and allow different applications to call the codegen command 

anyhow, as a starting solution i m more than happy, would the `spec` bit work off a raw.github url?

Reply:
> anyhow, as a starting solution i m more than happy, would the spec bit work off a raw.github URL?

Yeah, something like that. If you look at the generated file, it's the `gzip` (base64 encoded). I don't implement the endpoint to serve the spec just now. If we want to do it, we need to define it in the ReBAC admin spec as well.

Reply:
https://github.com/canonical/identity-platform-admin-ui/blob/81c8459c86f6f1aebb19d4b38d7662e368955fcd/rebac-admin-backend/internal/api/v1/types.go#L401